src/inspect_ai/tool/_tools/_web_browser/_web_browser.py

```diff
@@ -382,6 +397,58 @@ async def web_browser_cmd(cmd: str, *args: str) -> str:
                 f"web_browser output must contain either 'error' or 'web_at' field: {result.stdout}"
             )
 
+async def web_browser_cmd_local(cmd: str, *args: str) -> str:
+    session_flag = ""
+    browser_session = store().get(BROWSER_SESSION_ID, "")
+    
+    if not browser_session:
+        if not os.path.exists(LOCAL_WEB_CLIENT_NEW_SESSION):
+            raise RuntimeError("Local web browser new session script not found")
+            
+        process = await asyncio.create_subprocess_exec(
+            sys.executable,
+            LOCAL_WEB_CLIENT_NEW_SESSION,
+            stdout=asyncio.subprocess.PIPE,
+            stderr=asyncio.subprocess.PIPE
+        )
+        stdout, stderr = await process.communicate()
```

Comment:
```suggestion
        stdout, stderr = await process.communicate() # TODO see if we can import methods and call natively instead of running via subprocess
```